charts/astronomer/templates/config-syncer/config-syncer-cronjob.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  schedule: "{{ .Values.configSyncer.schedule }}"
+  schedule: {{ include "configSyncer.schedule" .}}
```

Comment:
Can we get a test for this? This is how we did it with the other adler32sum minute. <https://github.com/astronomer/astronomer/blob/c66c8f75e4/tests/chart_tests/test_houston_configmap.py#L124-L175>

Reply:
Done